incident/incident_test.go

```diff
@@ -475,13 +475,12 @@ func TestResponders_Validate(t *testing.T) {
 		{Type: User},
 	}
 	err = validateResponders(Responders)
-	assert.Equal(t, err.Error(), errors.New("For responder either name"+
-		" or id must be provided.").Error())
+	assert.Equal(t, err.Error(), errors.New("For responder type user either username or id must be provided.").Error())
 
 	Responders = []Responder{
 		{
-			Type: User,
-			Name: "cem",
+			Type:     User,
```

Comment:
nit: can we remove extra space

Reply:
go fmt is aligning the values, if you check the other examples,
```go
	Responders = []Responder{
		{
			Type: Team,
			Id:   "06",
		},
	}
```	
You'll see a similar indentation.